### DISCUSSION TITLE
Appear to be logged out on the QFieldCloud on QGIS Desktop

### DISCUSSION THREAD

**Discussion by enheduanna** [opengisch/QField#2628] [Q&A]
Hi 
I am functional in QGIS but definitely not wizard. I am setting up an archaeological field project and now want to test the GFieldCloud. I understand that I set up the project on QGIS Desktop as I would if I would use the traditional sync feature. I do see the cloud in the QGIS desktop interface but it says I am logged out and when I click on it - it will not allow me to log in. Do I have to create the project in the cloud first or should I be able to create the project in QGIS desktop and the push it to the cloud from where I can push it onto a device of mine and then started editing in the field and push it back to the cloud once I am done? 
Thank you
Stephanie #

**Comment by suricactus** [comment]
hi @enheduanna .

Can you give a little bit more details about your QFieldSync plugin version, QGIS version, Qt version and your operating system?

> it will not allow me to log in

What exactly happens? You don't see the login form? Or it shows "login failed"? 

Please give as much details.

Best,
Ivan

**Comment by suricactus** [comment]
I believe your issue might be similar to this one here. https://github.com/opengisch/QField/discussions/2732

**Comment by enheduanna** [comment]
Dear Ivan
Thank you very much but in my case the issue was that I had to download the
new version of QGIS and also the black developer app and then it worked. I
signed in online and it worked also on QGIS and on the mobile app. I am
using it now on several tablets and it works just fine.
Than you very much for reaching out.
Best wishes
Stephanie (aka enheduanna)

**Reply by suricactus** [reply]
Since you are there and you have opened a few issues on the github discussions page, please make sure to answer/update them if you are no longer experiencing the problems. :)

**Comment by enheduanna** [comment]
Noted with thanks - will do in the future.
Sorry for not realizing sooner.
